lsp-biome.el

```diff
@@ -25,10 +25,11 @@
 
 ;;; Commentary:
 
-;; 
+;;
 
 ;;; Code:
 
+(require 'cl-seq)
```

Comment:
Maybe this import is not strictly required, just in case; it provides `cl-find-if`.